### PR TITLE
fix: removed config fragment and moved the setting to lux meter settings

### LINKED ISSUE #1291: simplify setting option in lux meter
**Actual Behaviour**

the settings option in lux meter is confusing.

**Expected Behaviour**

The setting option should be removed and simple button with proper icons for different should be added so that user can understand easily.

**Steps to reproduce it**

LuxMeter Activity->Simplify menu options

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1369: remove the lux config fragment
**Actual Behaviour**

The lux config fragment in lux meter for setting the properties of lux meter.

**Expected Behaviour**

the lux meter setting should be moved to some other setting fragment and config fragment should be removed.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Avjeet**
Fixes #1369 
Fixes #1291 
**Changes**: 

1. Removed the lux meter config fragment
2. Moved the sensor configure to lux meter setting 
3. Renamed setting option in the overflow menu to lux meter setting
4. added select sensor and gain value card view in lux meter

**Screenshot/s for the changes**: 

lux meter setting

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:
[setting_updated.zip](https://github.com/fossasia/pslab-android/files/2296103/setting_updated.zip)

### CODE REVIEW

**Comment by cweitat** [comment]
Please fix codacy

**Comment by Avjeet** [comment]
@CloudyPadmal @harsh-2711 @abhinavraj23 if anyone has the light sensors then pls test the app with them and make sure the app is working as it was before this PR.

**Comment by Avjeet** [comment]
@cweitat fixed codacy issues (except the one with butterknife library)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -919,6 +919,11 @@
         <item>Built In</item>
         <item>BH1750</item>
         <item>TSL2561</item>
+    </string-array>\>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: There's an extra closing here

**Comment by Avjeet** [comment]
@CloudyPadmal apk updated

**Comment by Avjeet** [comment]
@CloudyPadmal @harsh-2711 @abhinavraj23 @cweitat plz review this soon

**Comment by CloudyPadmal** [comment]
I tested the apk

- The chart is not getting any reading updated.
- There was a crash when I click on one item from the list of logged data. (Probably because the chart is getting no data). The report is as follows;
```
java.lang.IndexOutOfBoundsException: Invalid index 0, size is 0
at java.util.ArrayList.throwIndexOutOfBoundsException(ArrayList.java:255)
at java.util.ArrayList.get(ArrayList.java:308)
at com.github.mikephil.charting.data.DataSet.getEntryForIndex(DataSet.java:286)
at com.github.mikephil.charting.utils.Transformer.generateTransformedValuesLine(Transformer.java:184)
at com.github.mikephil.charting.renderer.LineChartRenderer.drawValues(LineChartRenderer.java:547)
at com.github.mikephil.charting.charts.BarLineChartBase.onDraw(BarLineChartBase.java:264)
at android.view.View.draw(View.java:15204)
```

**Comment by Avjeet** [comment]
@CloudyPadmal Can u post a gif or something because in my device everything is working fine.

**Comment by cweitat** [comment]
When trying to playback the recorded data, the app crashes and goes to a white screen

**Comment by cweitat** [comment]
Also, when I spam the delete button to delete all the data, while the item was sliding up, I managed to clicked the delete button which resulted the app crashing as well. Please fix that problem.

**Comment by CloudyPadmal** [comment]
@Avjeet I've created a gif from the APK here. As you can see there is no data being shown in graph and that is why it's crashing when trying to open a saved record.

luxmetercrash

**Comment by Avjeet** [comment]
@CloudyPadmal could you please test the app again with some another device because it seems like maybe your sensor in the phone is not fetching any data and as you can recall I have shown this apk in the previous meeting and it was working perfectly.

**Comment by Avjeet** [comment]
@CloudyPadmal But on another note that is an issue that app should not store if nothing has been recorded to avoid crashes as you mentioned, I will resolve right away.

**Comment by cweitat** [comment]
@CloudyPadmal can you also check if the previous versions also had the same problem or just this version that is not plotting the data. If previous versions also had the same problem then have to look deeper

**Comment by CloudyPadmal** [comment]
@Avjeet as you can see the gauge is showing values. That narrows it down to an issue in plotting. If my device doesn't have a lux sensor I cannot get those values in gauge right?

**Comment by Avjeet** [comment]
@CloudyPadmal That gauge is only showing value because of the method 
` lightMeter.setWithTremble(true);`
which fluctuates the pin and value of gauge to provide physical gauge like experience, you can see in the image the threeboxes below doesnot show any value that means no value was fetched.

**Comment by CloudyPadmal** [comment]
@Avjeet I see! That clarifies it! Thank you!

I've installed a different lux meter app from playstore and then it starts fetching data. Is there any setting that we have to enable Lux sensor in device may be? Now when I tried to play the data from logged data list (By clicking the Play button from menu), app crashes giving this log;

```
Process: io.pslab, PID: 8076
android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
    at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:6264)
    at android.view.ViewRootImpl.invalidateChildInParent(ViewRootImpl.java:906)
    at android.view.ViewGroup.invalidateChild(ViewGroup.java:4638)
    at android.view.View.invalidateInternal(View.java:11766)
    at android.view.View.invalidate(View.java:11730)
    at android.view.View.invalidate(View.java:11714)
    at io.pslab.activity.SensorGraphViewActivity$2.run(SensorGraphViewActivity.java:285)
    at java.lang.Thread.run(Thread.java:818)
```

**Comment by cweitat** [comment]
@CloudyPadmal if no other issue can we merged this?

**Pull request merged**